lib/algos/list/tools.js

```diff
@@ -31,9 +34,29 @@ function inc(str) {
             String.fromCharCode(str.charCodeAt(str.length - 1) + 1)) : str;
 }
 
+function listingParamsMasterKeysV0ToV1(v0params) {
+    const v1params = Object.assign({}, v0params);
+    if (v0params.gt !== undefined) {
```

Comment:
Since you copied the v0 data here, it might be good to use v1params instead of v0 across the board.

Reply:
The idea is to check against what is provided from the input, because we are going to transform an input into an output, so I think `v0params.gt` makes sense.

Reply:
Right, just commenting that since you copy the input straight into the output on the first line. We could just check the output across the board since they are the same. Or just set the output to an empty object and merge the two at the end. Would have identical results. But just a preference thing here.